Pipfile

```diff
@@ -0,0 +1,28 @@
+[[source]]
+url = "https://pypi.python.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[[source]]
+url = "https://pypi.pacificclimate.org/simple"
+verify_ssl = true
+name = "pcic"
+
+[packages]
+python-dateutil = "*"
+cdo = "==1.5.3"
+nchelpers = "==5.5.6"
+Pint = "*"
+PyYAML = "*"
+
+[dev-packages]
+pytest = "*"
+pytest-mock = "*"
+black = "==21.7b0"
+
+[requires]
+python_version = "3.8"
+
+[packages.e1839a8]
```

Comment:
What is the significance of `e1839a8`?

Reply:
So that was an `install -e .` and I believe the significance of the number is the commit hash?

Reply:
[Nope, not a commit hash.](https://github.com/pacificclimate/climate-explorer-data-prep/commit/e1839a8). Maybe some other kind of hash (package hash? like the package verification SHAs?) created by `pipenv`? Anyway, not really an issue, except I hate weird mystery hex codes scattered like chocolate sprinkles across code.